react/features/chat/components/native/Chat.js

```diff
@@ -32,7 +32,9 @@ class Chat extends AbstractChat<Props> {
                 headerProps = {{
                     headerLabelKey: 'chat.title'
                 }}
-                modalId = { CHAT_VIEW_MODAL_ID }>
+                modalId = { CHAT_VIEW_MODAL_ID }
+                onClose = { this.props._onToggleChat }>
```

Comment:
This won't work, you need to pass a function which returns true for the modal to close, as it stands it will remain open.